pkg/frontend/init_db.go

```diff
@@ -597,6 +598,38 @@ func PrepareInitialDataForMoUser() [][]string {
 	return data
 }
 
+// DefineSchemaForMoUser decides the schema of the mo_table
+func DefineSchemaForMoIincrement() *CatalogSchema {
+	/*
+		mo_increment_columns schema
+		| Attribute |     Type     | Primary Key |             Note         |
+		| -------   | ------------ | ----------- | ------------------------ |
+		|   name    | varchar(770) |             | Name of the db_table_col |
+		|   num     |    int64     |             |   current index number   |
```

Comment:
The mysql has offset(default is 1) and increment (default is 1) for the auto_increment. Does the mo need these two things ?